glean-core/ios/Glean/Metrics/StringMetric.swift

```diff
@@ -0,0 +1,96 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+
+/// This implements the developer facing API for recording string metrics.
+///
+/// Instances of this class type are automatically generated by the parsers at build time,
+/// allowing developers to record values that were previously registered in the metrics.yaml file.
+///
+/// The string API only exposes the `StringMetricType.set` method, which takes care of validating the input
+/// data and making sure that limits are enforced.
+public class StringMetricType {
```

Comment:
Is there a bug for adding usage examples [to the docs](https://mozilla.github.io/glean/book/user/metrics/string.html)?

Reply:
I should probably just add the docs here.,

Reply:
I also created [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1581586) yesterday for this in general.